### PR TITLE
Fix Ukrainian ISO & some refactor for use plurals

### PR DESCRIPTION
**Pull request by alexdee2007**
Fix ISO code Ukrainian ua -> uk
Refactor some langs for use plural handler

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by alexdee2007** [comment]
I think it would be right to change other languages (hr, pl, ro) for use plural handler also...

File: src/utils/plural.js [hunk 1]
```diff
@@ -3,10 +3,51 @@ import i18n from '../i18n'
 // Pluralization l10n http://docs.translatehouse.org/projects/localization-guide/en/latest/l10n/pluralforms.html?id=l10n/pluralforms
 export function declOfNum (n, titles) {
   switch (i18n.name) {
```
Comment (rstoenescu) [inline src/utils/plural.js]: Where is i18n variable coming from?

**Review by rstoenescu — CHANGES_REQUESTED** [review]

File: src/utils/plural.js (hunk 1 above)
Comment (alexdee2007) [reply src/utils/plural.js]: import i18n from '../i18n'
in first line

**Comment by rstoenescu** [comment]
@alexdee2007 

Need some help to port this addition to the ukraining (and russian) translation. Can you help, pls?

```js
tree: {
    noNodes: 'No nodes available',
    noResults: 'No matching nodes found'
  }
```

Thanks!

**Comment by alexdee2007** [comment]
@rstoenescu 

Yes, of course. I'm going to create a PR...

**Comment by rstoenescu** [comment]
Thank you!!

**Pull request merged**